timbot.py

```diff
@@ -63,10 +62,7 @@ def uploadimage(path, title, text):
 
 
 def choose_lunchcation():
-    places = ['pauls', 'asian plus', 'moes', 'the 99', 'chilis']
```

Comment:
we need all this put back!

Reply:
Nope, it's not needed if we always choose the 99 as the lunch location.